src/components/Feedback/Feedback.js

```diff
@@ -0,0 +1,75 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import { settings } from 'carbon-components';
+import { Button } from 'carbon-components-react';
+import {
+  ModalHeader,
+  ModalBody,
+  ModalFooter,
+} from 'carbon-components-react/lib/components/ComposedModal';
+
+import HeaderMenuItem from '../HeaderMenuItem';
+
+const { prefix } = settings;
+
+class Feedback extends Component {
+
+  handleCloseModal = (closeModal) => {
+    closeModal();
+  };
+
+  render() {
+    return (
+      <HeaderMenuItem
+        preventCloseOnClickOutside
+        text="Submit an Idea"
+        iconName="idea"
+        className={`${prefix}--bmrg-feedback-container`}
+      >
+        {({ closeModal }) => {
+          return (
+            <>
+              <ModalHeader
+                title="Submit an Idea"
+                closeModal={() => this.handleCloseModal(closeModal)}
```

Comment:
I'm guessing this is a carry over from the other code? Can we simplify this by just passing the closeModal function. 
```suggestion
                closeModal={closeModal}
```

Reply:
I thought there's some sort of code styling that is required/suggested to be followed. The simple one looks better to me too.

Reply:
Cool yeah given that we used class components I would guess its pretty old code at this point. I probably didn't know what I was doing really.